build/scan-appdir.js

```diff
@@ -1,6 +1,5 @@
 'use strict';
 
```

Comment:
Because we output an appList value through stdout and assign to Makefile variable on this way. It's very trivial may confuse contributors.
I'd like to put some words / comments here to explain the tricky way we did in this script.